elixir/config/config.exs

```diff
@@ -64,7 +64,19 @@ config :domain, Domain.ChangeLogs.ReplicationConnection,
     resource_connections
     resources
     tokens
-  ]
+  ],
```

Comment:
I consolidated the replication config here because the compile-time checks work a bit better than inside the `__using__` macro, and also for consistency.